src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CSharpStatementBlockOnTypeFormattingTest.cs

```diff
@@ -14,7 +14,7 @@ public async Task CloseCurly_IfBlock_SingleLine()
             await RunOnTypeFormattingTestAsync(
 input: @"
 @{
- if(true){|}|
+ if(true){$$}
```

Comment:
To me this is strange affinity for a position test, but it matches the previous behavior.